src/ui/templates/insurance/declarations/anti-bribery.njk

```diff
@@ -45,7 +45,7 @@
   </div>
 
   {{ expandableContent.render({
-    CONTENT: CONTENT_STRINGS.EXPANDABLE
+    CONTENT: CONTENT_STRINGS.VERSIONS[0].EXPANDABLE
```

Comment:
Should this be in this PR? As it's not just splitting up the content strings

Reply:
Tests fail without it, is temporary